sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1138,6 +1138,11 @@
                 <!-- Azure-core needs to open the test entities to Jackson for unit tests to operate -->
                 --add-opens com.azure.core/com.azure.core.implementation.entities=com.fasterxml.jackson.databind
                 --add-opens com.azure.core/com.azure.core.implementation.entities=ALL-UNNAMED
+				
```

Comment:
IIRC, we were trying to push this into the actual child pom.xml than bloat the parent one. @alzimmermsft, is this correct?

Reply:
Yes, we should begin removing library specific configurations from the parent POM. The current configuration results in a lot of warnings from being unable to find modules that exist outside of the project being ran.